intel-sgx/ppid-retrieval-tool/Readme.md

```diff
@@ -0,0 +1,62 @@
+# PPID Decryption Tool
+
+This tool provides functionality to decrypt the **Platform Provisioning ID (PPID)** from the **Provisioning Certification Enclave (PCE)**.
+
+## Overview
+
+The tool is inspired by Intel's **PCKRetrievalTool**, which is used to retrieve platform-specific information from the PCE. However, Intel’s tool returns only an **encrypted version of the PPID**. This utility extends the functionality by enabling decryption of the PPID, allowing access to the data in an unencrypted form.
```

Comment:
I think it worth clarify that:
The PCE only provides the API to get the encrypted PPID with given RSA 3072 public key.
Intel's `PCKRetrievalTool` use [id_enclave](https://github.com/fortanix/SGXDataCenterAttestationPrimitives/tree/main/QuoteGeneration/quote_wrapper/quote/id_enclave) to load a their public key to get the encrypted PPID. So, we cannot decrypt.

This tools flow the similar process but using our own RSA key to get the encrypted PPID, so we can decrypt since we own the private key.